Added homework file 'Exercise_1.1.ipynb' via upload

ninakuni opened the pull request:
Hi Nils, my homework is enclosed in file 'Exercise_1.1.ipynb
Please let me know if something went wrong with the upload and if there's something wrong, missing or whatever :) 
Thanks, Nina

ninakuni committed:
Add files via upload

nilsvu commented:
Hi @ninakuni, this looks good 👍 A few things I noticed:
- You integrate only to $t=3 M$, instead of $t=10 M$
- Your light rays start at an initial coordinate distance $0 <= r0 <= 1 M$ from the black hole, which is why they all fall in (their coordinate distance decreases with time). Try starting the light ray outside the horizon $r0 >2 M$ :)
- The coordinate $r$ is only defined for positive values, so you could stop the integration once $r$ becomes negative.

ninakuni committed:
Update Exercise_1.1.ipynb

Hi Nils,
tried to fix that r0 > 2M but I think adapting the np.linspace(...) command isn't the right approach..

To stop integration when r becomes negative, I tried to implement if r == 0:  and so on, but that didn't really work.  Maybe you could give me another hint

Nina

nilsvu commented:
Hi Nina,
- The way you changed `np.linspace` looks good - see how the curves go outward to increasing $r$ when you start outside of $r0=2M$? If you choose $r0$ between 1 and 3, for example, then you'll even see some curves falling in and some going out.
- To prevent $r$ from going negative, you could try to change the condition for the `while` loop like this:
  ```python
  while r.successful() and r.t < 10. and r_history[-1] > 0:
      ...
  ```
  The `r_history[-1]` picks the last element from the list, which is the one that was just added in the previous iteration of the loop.

nilsvu commented:
5/5 points. 💯
